rpg.py

```diff
@@ -163,6 +158,10 @@ def run(self):
 
 
 if __name__ == '__main__':
+    print('What is your name?')
+    name = ''
+    while name == '':
```

Comment:
Aqua's turn
Who do you attack?
Krillin
Yamcha
>Krillin
Aqua attacked Krillin for 4 damage
Aqua's turn
Who do you attack?
Krillin
Yamcha
>Krillin
Aqua attacked Krillin for 4 damage
Krillin's turn
Tyler's turn
Who do you attack?
Yamcha
>Yamvha
>Yamcha
Tyler attacked Yamcha for 3 damage
Aqua's turn
Who do you attack?
Yamcha
>Yamcha
Aqua attacked Yamcha for 4 damage
Yamcha's turn
Krillin's turn
Tyler's turn
Who do you attack?

this isn't an error message, but it's basically saying, Krillin and Yamcha are both dead but it's still Tyler's turn

Reply:
This really doesn't help....

Are you running into an `exception`? As in the code actually fails. Or is it working, but not giving the correct behavior?